src/app/web-app.component.ts

```diff
@@ -19,13 +19,21 @@ import { Logger } from './core/logger/logger.service';
 import { I18nService } from './core/i18n/i18n.service';
 import { ThemeStorageService } from './shared/theme-picker/theme-storage.service';
 import { AlertService } from './core/alert/alert.service';
+import { KeyboardShortcutsService } from './help/keyboard-shortcuts/keyboard-shortcuts.service';
+import { AuthenticationService } from './core/authentication/authentication.service';
+import { SearchService } from './shared/search/search.service';
 
 /** Custom Models */
 import { Alert } from './core/alert/alert.model';
 
 /** Initialize Logger */
 const log = new Logger('MifosX');
 
+/**
+ * Empty interface to get the key bindings and then remove unlisten from listeners list after the user is out of zone.
```

Comment:
The comment is unclear, you could rename the variable to `unlistenKeyboardShortcuts` or something relevant which might help someone to understand this better. Also, specify what is `zone`.
You can take the comment to multiple lines if its becoming to long.

Reply:
I've removed the unlisten part as it was making it more confusing to understand.